### ISSUE TITLE
Add `aria-required` to symptoms question in test card form

### ISSUE THREAD

**Issue by emyl3** [CDCgov/prime-simplereport#8552]
## Area or Component

Conduct tests -> Test Card -> "Select any symptoms the patient is experiencing" (for all diseases that ask this question) and "When did the patient's symptoms start?" (for non STI diseases AoE questions)

## Quick Summary

For some conditionally required fields, when the user selects "Yes" for "Is the patient currently experiencing or showing signs of symptoms?", it is not clear that the following AoE questions are required:
- "When did the patient's symptoms start?" (for non-STI diseases)
- "Select any symptoms that the patient is experiencing" (for all diseases that ask this)

### How to Replicate
See above

## Accessibility Violation

[aria-required-missing
](https://docs.deque.com/issue-help/1.0.0/en/aria-required-missing)

## Action Requested

Ensure those conditionally required fields have both a visual and programmatic way for users to know those fields are required. (with an * and a `aria-required` or `required` attribute.

## Screenshots (optional)

<img width="1689" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBelFUQ1E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--59e9af99390684d2b744c4e4b3a7c1895a136707/Screenshot%202025-02-28%20at%2015.10.59.png" alt="Screenshot 2025-02-28 at 15.10.59.png" />

<img width="1000" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBellUQ1E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--e71b6340582af371476ccc66cd5a8f874abdf441/Screenshot%202025-02-28%20at%2015.18.25.png" alt="Screenshot 2025-02-28 at 15.18.25.png" />

## Source Code (optional)

[which file or line is the issue attached to?]: #
